core/dmux.go

```diff
@@ -220,7 +220,7 @@ func getStopMsg() ControlMsg {
 	return c
 }
 
-func (d *Dmux) runWithSideline(source Source, sink Sink, sidelineImpl sideline_module.CheckMessageSideline) {
+func (d *Dmux) runWithSideline(source Source, sink Sink, sidelineImpl sideline_module.CheckMessageSideline, enableDebugLog bool) {
```

Comment:
Instead of adding another parameter, can we make it a struct of optional params?

Reply:
have added optional params, please check now